less/index.less

```diff
@@ -54,6 +54,65 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	
 }
 
+// ======== === === == =====Variables
```

Comment:
good job with making the comment to help point out where your variables are (same with the mixins on line 104). You demonstrated you understand the purpose and point of the lesson and why variable and mixins matter